examples/armv4t/test_bin/.gdbinit

```diff
@@ -1,2 +1 @@
-file test.elf
```

Comment:
this should not have been comitted.

Reply:
This is intentional. Now gdb will request the file with `exec-file` and `vFile:pread` automatically. If we use `file` to load it, `pread` won't be invoked.

Before:
```
(gdb) file test.elf
Reading symbols from test.elf...
(gdb) target extended-remote :9001
Remote debugging using :9001
main () at test.c:1
1       int main() {
```
After:
```
(gdb) target extended-remote :9001
Remote debugging using :9001
Reading /test.elf from remote target...
warning: File transfers from remote targets can be slow. Use "set sysroot" to access files locally instead.
Reading /test.elf from remote target...
Reading symbols from target:/test.elf...
main () at test.c:1
1       int main() {
```